get_BIDS_sub.py

```diff
@@ -0,0 +1,118 @@
+import sys
+import os
+import argparse
+import json
+import subprocess
+import nibabel as nib
+import glob
+import numpy
+
+def get_parser():
+    # Mandatory arguments
+    parser = argparse.ArgumentParser(
+        description=" ",
+        epilog="EXAMPLES:\n",
+        add_help=None,
+        prog=os.path.basename(__file__).strip('.py'))
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        '-path',
+        required=True,
+        help="path to BIDS Data",
+    )
+    mandatoryArguments.add_argument(

```

Comment:
`constraint`, `ope`, `value` are great.
Possible avenue of improvement: add possibility to have multiple constraints (separated by commas?).
Could also use a config file to specify those.

Reply:
Is there a reason for requiring them as mandatory? I would recommend to change for optional.

Reply:
Not really, in my mind there was another solution to get all file but I just realized that's will not work with the 'manual labeling.py' script. (ls > file.txt would get you all the subject but not image file) will fix